docs/blog/v1.md

```diff
@@ -0,0 +1,187 @@
+[MDX Blog](/blog)
+
+# MDX goes stable
+
+It’s been a year and a half since the first MDX commit and a year since MDX was first announced at
+ZEIT Day.  MDX is a radical paradigm shift in how to write immersive content using components.  It’s
+an open, [authorable format](https://johno.com/authorable-format) that makes it _fun_ to write again.
+
+Over the last year we’ve been working on numerous bug fixes, new features, better parsing, and integration
```

Comment:
```suggestion
Since announcing MDX we’ve been working on numerous bug fixes, new features, better parsing, and integration
```

(reference back to earlier paragraph)